physics/src/IceGrowth.cpp

```diff
@@ -181,24 +201,25 @@ void IceGrowth::lateralIceSpread(size_t i, const TimestepTime& tstep)
         // Note that the cell-averaged hice0 is converted to a ice averaged value
         iLateral->melt(tstep, hice0[i], hsnow[i], deltaHi[i], cice[i], qow[i], deltaCMelt[i]);
     }
-    double deltaC = deltaCFreeze[i] + deltaCMelt[i];
-    cice[i] += deltaC;
-    if (cice[i] >= MinimumIce::concentration()) {
+    deltaCIce[i] = deltaCFreeze[i] + deltaCMelt[i];
+    cice[i] += deltaCIce[i];
+    if (cice[i] >= IceMinima::cMin) {
         // The updated ice thickness must conserve volume
-        updateThickness(hice[i], cice[i], deltaC, newice[i]);
-        if (deltaC < 0) {
+        updateThickness(hice[i], cice[i], deltaCIce[i], newice[i]);
+        if (deltaCIce[i] < 0) {
             // Snow is lost if the concentration decreases, and energy is returned to the ocean
-            qow[i] -= deltaC * hsnow[i] * Water::Lf * Ice::rhoSnow / tstep.step;
+            qow[i] -= deltaCIce[i] * hsnow[i] * Water::Lf * Ice::rhoSnow / tstep.step;
         } else {
             // Update snow thickness. Currently no new snow is implemented
-            updateThickness(hsnow[i], cice[i], deltaC, 0);
+            updateThickness(hsnow[i], cice[i], deltaCIce[i], 0);
         }
     }
 }
 
 void IceGrowth::applyLimits(size_t i, const TimestepTime& tstep)
 {
-    if (cice[i] < MinimumIce::concentration() || hice[i] < MinimumIce::thickness()) {
+    if ((0. < cice[i] && cice[i] < IceMinima::cMin)
```

Comment:
Let's keep one format and saying cice is better than zero (a.k.a. it exists) is simpler.

Obvs this logic is slightly different so re-jig as needed. 

```suggestion
    if ((cice[i] > 0 && cice[i] < IceMinima::cMin)
```